qkernel/src/kernel_def.rs

```diff
@@ -208,24 +208,77 @@ pub fn Invlpg(addr: u64) {
     }
 }
 
+
+
+#[derive(Debug, Copy, Clone, Default)]
+#[repr(C)]
+pub struct SharePara {
+    pub para1: u64,
+    pub para2: u64,
+    pub para3: u64,
+    pub para4: u64,
+}
+
+pub const SHAREPARA_COUNT: u64 = MemoryDef::PAGE_SIZE/core::mem::size_of::<SharePara>() as u64;
```

Comment:
This seems strange. Shouldn't it be four?

Reply:
I leave a page for shared parameters, the count is the maximum number of SharePara structs in this page.